lib/nx_uncompr.c

```diff
@@ -99,8 +102,11 @@ int uncompress(Bytef *dest, uLongf *destLen, const Bytef *source, uLong sourceLe
 #else
 	int rc=0;
 
-	if(nx_config.gzip_selector == GZIP_MIX){
-		rc = sw_uncompress(dest, destLen, source, sourceLen);
```

Comment:
This may be unrelated to this PR, but for `GZIP_MIX`, don't we also need to choose based on `nx_ratio` here? In this case `GZIP_MIX` and `GZIP_MIX2` are the same thing.

Reply:
This is another problem with the description. The reasoning behind `GZIP_MIX` decompression is the same as `GZIP_MIX2`, as the decompression throughput is usually higher on software there is not much reason to use the ratio for decompression as well, but I'm not against it.

Reply:
Fair enough. As I said in the other comment, improving the description of `GZIP_MIX` is enough IMO.